docker-environment-driver/src/main/resources/dockerConfiguration/Dockerfile

```diff
@@ -19,13 +20,23 @@ RUN echo "export VISIBLE=now" >> /etc/profile
 #RUN chown jenkins /usr/sbin/sshd
 
 #Install maven
-ADD http://mirror.hosting90.cz/apache/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz /root/maven.tar.gz
-RUN mkdir /usr/apache-maven/
-RUN tar -xzvf /root/maven.tar.gz --directory=/usr/apache-maven
-RUN echo "export M2_HOME=/usr/apache-maven/apache-maven-3.2.5" >> /etc/profile
-RUN echo "export PATH=$PATH:/usr/apache-maven/apache-maven-3.2.5/bin" >> /etc/profile
+# set installed Maven version you can easily change it later
+ENV MAVEN_VERSION 3.2.5
+RUN curl -sSL http://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xzf - -C /usr/share \
+&& mv /usr/share/apache-maven-$MAVEN_VERSION /usr/share/maven \
+&& ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+ 
+RUN echo "export M2_HOME=/usr/share/maven" >> /etc/profile
```

Comment:
Sorry I didn't make myself clear - I was referring to `export M2_HOME=/usr/share/maven`. Why shouldn't we use simply `~/.m2/repository`?